### ISSUE TITLE
Error when loading DICOM images, image with noise when using Arch Linux

### ISSUE THREAD

**Issue by MuriloSchaefer** [FNNDSC/ami#277]
Hi,

I'm using AMI.js with OHIF Viewer, and it was working fine in Ubuntu, but when I'm using Arch Linux, every image loaded came with some noise (see Figure 1), then I zoomed to see what is this noise, and discover that is like the image has been sliced in many rows, and between this rows there a space (Figure 2). The Figure is from lesson 1 example.

Figure 1
![fig1](https://user-images.githubusercontent.com/14366202/39138162-b668e660-46f5-11e8-8765-14ebf1dedc24.png)

Figure 2
![fig2](https://user-images.githubusercontent.com/14366202/39138163-b6903134-46f5-11e8-8290-58cdc87f31a2.png)

About the machine: 
I'm using an Intel i7, 8Gb of RAM, Google Chrome version 66 and Firefox 59 (both presented the same issue). When I run the script `$uname -a` I get:
`Linux Schaefer 4.16.3-1-ARCH #1 SMP PREEMPT Thu Apr 19 09:17:56 UTC 2018 x86_64 GNU/Linux`   

I believed that was an error with the three.js lib, but I can't reproduce the error with their examples, so I decided to post it here to see if I get some possible cause of this error.

Thank you,
Murilo Schaefer

**Comment by NicolasRannou** [comment]
Interesting…

What is the same machine working fine in Ubuntu or a different machine?

**Comment by MuriloSchaefer** [comment]
It's a different machine, but when I use the Windows 10 installed on the same machine from Arch, it works fine, using the same version of Google Chrome and Firefox.

**Comment by MuriloSchaefer** [comment]
ADD: 
When I open the lesson with the console opened, some warnings came up:

`THREE.WebGLShader: gl.getShaderInfoLog() vertex 0:1(12): warning: extension 'GL_ARB_gpu_shader5' unsupported in vertex shader`

`THREE.WebGLShader: gl.getShaderInfoLog() fragment 0:1(12): warning: extension 'GL_ARB_gpu_shader5' unsupported in fragment shader`

This message isn't shown in Google Chrome, only in Firefox. There are some topics in three.js repository about it and apparently, this is a Firefox issue, so I don't know if it's related if the bug.

**Comment by NicolasRannou** [comment]
Hmm yes, this one is tough, I suspect some driver / graphic-card specific thing. Could you find a THREE example not working using the "problematic machine"?

**Comment by NicolasRannou** [comment]
Yes I think that may be a precision issue in the shaders, that is graphic card specific. That is a bug in AMI but a tough one to track down :)

**Comment by NicolasRannou** [comment]
It is tough - do you have experience with WebGL/ OpenGL?

**Comment by MuriloSchaefer** [comment]
So, no, just the basics
